.gitlab-ci.yml

```diff
@@ -74,6 +85,21 @@ build:coreneuron:intel:
     - .spack_build
     - .spack_intel
 
+build:coreneuron+nmodl:gpu:
+  variables:
+    SPACK_PACKAGE: coreneuron
+    # +report pulls in a lot of dependencies and the tests fail.
+    # See https://github.com/BlueBrain/CoreNeuron/issues/518 re: build_type
+    SPACK_PACKAGE_SPEC: +nmodl+gpu+tests~legacy-unit~report build_type=RelWithDebInfo
+    SPACK_PACKAGE_DEPENDENCIES: ^bison%gcc^flex%gcc^eigen%gcc
```

Comment:
I think this line can be dropped (its value will be overridden by the `dotenv` artifacts from the `build:nmodl:gpu` job).
```suggestion
```